src/lib/tools.js

```diff
@@ -107,8 +107,19 @@
 		 */
 		updateCopyrights: function( targetFile ) {
 			var extension = CKBuilder.io.getExtension( targetFile.getName() ),
-				bomExtensions = { asp: 1, js: 1 };
+				bomExtensions = { asp: 1, js: 1 },
+				copyright;
+
+			if ( CKBuilder.options.lts ) {
+				copyright = 'CKEditor 4 LTS ("Long Term Support") is available under the terms of the Extended Support Model.';
+			} else if ( CKBuilder.options.commercial ) {
+				copyright = 'This software is covered by CKEditor Commercial License. Usage without proper license is prohibited.';
+			} else {
+				// Copyright only needs to be updated for LTS or commercial license.
+				return;
+			}
 
+			// `lineEndings` array includes all filetypes supported by CKEditor 4.

```

Comment:
It seems like this method is only called for files that had updated line endings, for example language files. I'm not sure if that's the design decision or some oversight - but the binaries of CKEditor 4 should have properly updated license headers for LTS, so I didn't change the original behavior of this script (I've only added support for LTS).